app/components/vistk-scatterplot.js

```diff
@@ -179,15 +179,27 @@ export default Ember.Component.extend({
     if(this.get('dataType') === 'industries') { return 'employment'; }
   }),
   didInsertElement: function() {
-    $.getJSON(`${apiURL}/data/location?level=department`).then((response) => {
```

Comment:
@QuinnLee Nooo worries. I tried to think about how to pull this out of vistk-scatterplot and really just pass in another parameter to the component to draw a line at a specific eci line but I couldn't really come up with a good way, because it looks like all the components have the same set of parameters so if I just add eciValue to this, then it looks awkward in, say, treemaps, so I left it as is.